generator/rs/lib.tmpl.rs

```diff
@@ -1,4 +1,6 @@
 #![deny(clippy::all)]
+// @todo - properly fix this clippy issue
+#![allow(clippy::zero_repeat_side_effects)]
```

Comment:
meh, I think this is a napi issue causing false positive in clippy

Reply:
Yup I agreed, let's see if the problem disappear in a future napi-rs release, I opened an issue to not forget that 
cf. https://github.com/ircam-ismm/node-web-audio-api/issues/137